boot/bootutil/src/encrypted.c

```diff
@@ -15,7 +15,11 @@
 #if defined(MCUBOOT_ENCRYPT_RSA)
 #include "mbedtls/rsa.h"
 #if MBEDTLS_VERSION_NUMBER >= 0x03000000
+#ifdef __ZEPHYR__
+#include "mbedtls/../../library/rsa_alt_helpers.h"
```

Comment:
Hmm, not sure I like this. Is it possible to add something to the Cmake file regarding this file to specifically add to the include path but only when compiling this file. (maybe set_source_files_properties)?

Reply:
@tejlmand Can you help with that?

Reply:
Following syntax works fine indeed.
``` Cmake
  set_source_files_properties(${BOOT_DIR}/bootutil/src/encrypted.c
  PROPERTIES
  INCLUDE_DIRECTORIES ${ZEPHYR_BASE}/../modules/crypto/mbedtls/library)
```